azure/template.json

```diff
@@ -0,0 +1,183 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceName": {
+            "type": "string"
+        },
+        "appServicePlanResourceGroup": {
+            "type": "string"
+        },
+        "customHostName": {
+            "type": "string"
```

Comment:
should have a default value property that is set to empty

defaultValue: ""

Reply:
Don't think this is needed as the master template already has a default value for it

https://github.com/SkillsFundingAgency/das-platform-building-blocks/blob/master/templates/app-service.json#L33